react-testing/src/API/openLibrary.org/index.ts

```diff
@@ -0,0 +1,82 @@
+import FetchHandler from '../../utils/FetchHandler/FetchHandler.ts'
```

Comment:
please, consider to stop using this code. It seems like this is a library you could bring from `npm install`.